### ISSUE TITLE
How to  reproducing the zero-shot  result   49.38% on ModelNet40 ?

### ISSUE THREAD

**Issue by TangYuan96** [tyhuang0428/CLIP2Point#3]
First of all, thanks for sharing your outstanding work

I try to train zero-shot (python pretraining. py) on ModelNet40Align and ModelNet40Ply,

the results can only reach 36.71% and 32.70%

But the result in the paper is 49.38%.

Can you retrain and get 49.38% results?  What should I pay attention to during the training

Thanks!

**Comment by tyhuang0428** [comment]
You said that you use the command "python pretraining. py", while I am not sure whether you are reproducing zero-shot classification or the image-depth pre-training. 
If you mean zero-shot, you can use the following command,
```
python zeroshot.py --ckpt [pre-trained_ckpt_path]
```
Otherwise, your claimed results (36.71% and 32.70%) may come from the validation set of our pre-training, which is a little bit different from our zero-shot setting. The accuracy of the validation set can reach 42.83% during our pre-training. And we find that the batch size can significantly affect the pre-training.
See if these can help you.

**Comment by tyhuang0428** [comment]
We found a bug in the pre-training code, and have already fixed it in the latest commit. We wrongly rotate the CAD models in ShapeNet when rendering depth maps. Rotation is needed only in downstream tasks.

Sorry for the confusion. Please let us know if we can assist with anything else.

**Comment by Belta911** [comment]
How can I reproduce the zero-shot reasoning on scanobjectnn to achieve an accuracy of 35.46, when I can only achieve 13%? And what changes do I need to make to the code?
